### ISSUE TITLE
Disabling antialias for text rendering

### ISSUE THREAD

**Issue by ne0ndrag0n** [memononen/nanovg#478]
We need nice, crisp font rendering for preformatted text in our game console. Is it possible to disable font smoothing/antialiasing for a font draw call?

**Comment by memononen** [comment]
No. But there are some pixel TTF/OTF fonts out there, which should work with NanoVG if you choose right font size.
